### ISSUE TITLE
Unable to build with CocoaPods could not find compatible versions for pod "macos_window_utils" error.

### ISSUE THREAD

**Issue by TakaTeke** [macosui/macos_ui#478]
Unable to build with CocoaPods could not find compatible versions for pod "macos_window_utils" error.

The version of macos_ui is 2.0.0.

Here are the error details:

[!] CocoaPods could not find compatible versions for pod "macos_window_utils":
  In Podfile:
    macos_window_utils (from `Flutter/ephemeral/.symlinks/plugins/macos_window_utils/macos`)

Specs satisfying the `macos_window_utils (from `Flutter/ephemeral/.symlinks/plugins/macos_window_utils/macos`)` dependency were found, but they required a higher minimum deployment target.

Please let me know if you have any good ideas.

Thank you.

**Comment by TakaTeke** [comment]
I was able to build after changing macOS > Podfile as follows.
platform : osx, '10.15'

thank you very much.

**Comment by Adrian-Samoticha** [comment]
I assume you’d had it set to

```podspec
platform :osx, '10.14.6'
```

before, when it didn’t work? If it doesn’t work this way I might look into updating the documentation.

**Comment by GroovinChip** [comment]
@TakaTeke are you still encountering this issue? If not, let's close this.

**Comment by rivasdiaz** [comment]
I just saw this error too, following the Getting Started > Manual Installation instructions in a clean new app.

The guide suggest it will add `macos_ui: ^1.2.0` but it adds `macos_ui: ^2.0.2` which makes sense, but the new version depends on `macos_window_utils` which seems to have a minimum macOS requirement of 10.14.6 while the default generated app has a minimum macOS requirement of 10.14. After seeing this issue and looking for more details in the readme, I found the instructions on the "Modern window look" section. I think it'll be good to add those instructions to the guide too.
